src/lib/components/common/stories/Button.stories.svelte

```diff
@@ -0,0 +1,49 @@
+<script context="module" lang="ts">
+  import { Story, Template } from "@storybook/addon-svelte-csf";
+  import { PlusCircle16 } from "svelte-octicons";
+  import Button from "../Button.svelte";
+
+  export const meta = {
+    title: "Components / Common / Button",
+    component: Button,
+    tags: ["autodocs"],
+    parameters: { layout: "centered" },
+  };
+</script>
+
+<Template let:args>
+  <Button {...args}></Button>
+</Template>
+
+<Story name="default" />
+
+<Story name="link" args={{ href: "https://www.documentcloud.org" }} />
```

Comment:
Unit or storybook interaction test asserting component is an `a` in this condition would be great to have here